sdks/java/extensions/gcp-core/src/main/java/org/apache/beam/sdk/extensions/gcp/options/CloudResourceManagerOptions.java

```diff
@@ -33,14 +29,4 @@
     + "https://cloud.google.com/resource-manager/ for details on CloudResourceManager.")
```

Comment:
I kept this as a marker interface incase we do add transport level options for CRM.
It also gives clarity as to what pipeline options it relies on.